packages/lesswrong/server/callbacks/sunshineCallbackUtils.tsx

```diff
@@ -66,6 +66,18 @@ export function isRecentlyDownvotedContent(voteableItems: (DbComment | DbPost)[]
   return downvotedItemCount >= downvotedItemCountThreshold;
 }
 
+function isActiveNegativeKarmaUser(user: DbUser, voteableItems: (DbComment | DbPost)[]) {
+    // Not enough engagement to make a judgment
+    if (voteableItems.length < 5) return false;
+
+    const oneWeekAgo = moment().subtract(7, 'days').toDate();
```

Comment:
Possibly we want a longer timeframe here for how recent a user's content needs to be than we use for the other rules?

Reply:
I personally would have used "a month" for all the similar auto-mod flags, since I think "a month ago" is still pretty recent and means they're reasonably likely to come back. But, also think a week is fine.

Reply:
I think a month makes sense for this rule, at least